### ISSUE TITLE
cdi_config flaky unit tests

### ISSUE THREAD

**Issue by igoihman** [kubevirt/containerized-data-importer#716]
**Is this a BUG REPORT or FEATURE REQUEST?**:

 /kind bug

**What happened**:
`TestCreatesIngressDifferentService` is randomly failing on Travis.

```
    config-controller_test.go:172: 2 unexpected actions: [{ActionImpl:{Namespace: Verb:list Resource:{Group:cdi.kubevirt.io Version:v1alpha1 Resource:cdiconfigs} Subresource:} Kind:cdi.kubevirt.io/v1alpha1, Kind=CDIConfig Name: ListRestrictions:{Labels: Fields:}} {ActionImpl:{Namespace: Verb:watch Resource:{Group:cdi.kubevirt.io Version:v1alpha1 Resource:cdiconfigs} Subresource:} WatchRestrictions:{Labels: Fields: ResourceVersion:}}]
    config-controller_test.go:187: 2 unexpected actions: [{ActionImpl:{Namespace: Verb:list Resource:{Group:route.openshift.io Version:v1 Resource:routes} Subresource:} Kind:route.openshift.io/v1, Kind=Route Name: ListRestrictions:{Labels: Fields:}} {ActionImpl:{Namespace: Verb:watch Resource:{Group:route.openshift.io Version:v1 Resource:routes} Subresource:} WatchRestrictions:{Labels: Fields: ResourceVersion:}}]
    config-controller_test.go:202: 2 unexpected actions: [{ActionImpl:{Namespace: Verb:list Resource:{Group:extensions Version:v1beta1 Resource:ingresses} Subresource:} Kind:extensions/v1beta1, Kind=Ingress Name: ListRestrictions:{Labels: Fields:}} {ActionImpl:{Namespace: Verb:watch Resource:{Group:extensions Version:v1beta1 Resource:ingresses} Subresource:} WatchRestrictions:{Labels: Fields: ResourceVersion:}}]
```

**What you expected to happen**:
Test should be successful

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- CDI version (use `kubectl get deployments cdi-deployment -o yaml`):
1.6
- Kubernetes version (use `kubectl version`):
1.11.1
- Cloud provider or hardware configuration:
- Install tools:
- Others:
Travis system

**Comment by mhenriks** [comment]
@igoihman this is still a problem I see quite frequently.

**Comment by mhenriks** [comment]
And it's not just the test mentioned above that fails.

**Comment by igoihman** [comment]
@mhenriks I will work on a fix

**Comment by awels** [comment]
Fixed by #1034
